### ISSUE TITLE
cannot @apply step class

### ISSUE THREAD

**Issue by manmen-mi** [saadeghi/daisyui#346]
daisyui version: `1.17.1`

Dear daisyUI team,

I'm using scss(in .vue file) like:
```scss
<style lang="scss" scoped>
.step__item {
  @apply relative step flex flex-col items-center justify-start w-1/3 min-w-[230px] text-sm;
}
~~~
```

and i got error `.step` class is not defined.
A change in this line seems to be the cause.

https://github.com/saadeghi/daisyui/blob/40c2d4a63cbf1db4cdbc21b4ee28facf8dee7301/src/components/styled/steps.css#L4

is this change permanently?

(sorry for my english, I'm not native)

**Comment by saadeghi** [comment]
Yes, it is permanent.
I had to use `.steps [class~="step"]` instead of `.steps .step` because `steps` needs to be responsive and there's [a limitation with Tailwind plugins](https://github.com/tailwindlabs/tailwindcss/issues/3361#issuecomment-927879795) to specify which class should get responsive prefixes. this is the only way for now.

So I think you should't use `@apply step` anymore.  
Please let me know if you have any questions.

**Comment by saadeghi** [comment]
This is fixed on version `2.0`
